src/shared/bootspec.c

```diff
@@ -1905,6 +1908,8 @@ int show_boot_entry(
                 boot_entry_file_list("efi", e->root, e->efi, &status);
         if (e->uki)
                 boot_entry_file_list("uki", e->root, e->uki, &status);
+        if (e->uki)
```

Comment:
Is the condition intentional or copy-paste error? If intentional, please adjust the printf with strna() or so.